src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java

```diff
@@ -57,6 +61,16 @@ public ResponseDefinition transform(final Request request,
                       context.put("requestAbsoluteUrl", request.getAbsoluteUrl());
                       context.put("requestUrl", request.getUrl());
                       context.put("requestMethod", request.getMethod());
+					  context.put("numberTool", new NumberTool());
+					  if (Objects.nonNull(parameters) && parameters.containsKey("query")) {
+					      String[] queryKeys = parameters.getString("query").split(",");
+                          for (String queryKey: queryKeys) {
+                              QueryParameter value = request.queryParameter(queryKey);
```

Comment:
I see that the intention of this pull is to provide the velocity context some aware of the query/path parameters of the request. I agree this could be useful. I'm going to add this as a feature request. This tool is in need of a new version anyway.